### ISSUE TITLE
Document how to use the flag --configuration-file-path

### ISSUE THREAD

**Issue by LaurenceChau** [dotnet/dotnet-monitor#2605]
### Documentation Request

Can not use the flag configuration-file-path in dotnet-monitor

I try to use the `--configuration-file-path` to specify the configuration file in dotnet-monitor command. However dotnet-monitor can not recognize the flag.
```
PS C:\Users\laurence> dotnet monitor collect --configuration-file-path C:\ProgramData\dotnet-monitor\settings.json 
Unrecognized command or argument '--configuration-file-path'.
Unrecognized command or argument 'C:\ProgramData\dotnet-monitor\settings.json'.

Description:
  Monitor logs and metrics in a .NET application send the results to a chosen destination.

Usage:
  dotnet-monitor collect [options]

Options:
  -u, --urls <urls>                   Bindings for the REST api. [default: https://localhost:52323]
  --metricUrls <metricUrls>           Bindings for metrics api. [default: http://localhost:52325]
  -m, --metrics                       Enable publishing of metrics [default: True]
  --diagnostic-port <diagnosticPort>  The fully qualified path and filename of the diagnostic port to which runtime instances can connect.
  --no-auth                           Turn off authentication. [default: False]
  --temp-apikey                       Generates a new MonitorApiKey for each launch of the process. [default: False]
  --no-http-egress                    Turn off HTTP response egress [default: False]
  -?, -h, --help                      Show help and usage information

```

### Previous documentation

The configuration mentioned the flag `--configuration-file-path`
https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration.md#configuration-sources

**Comment by jander-msft** [comment]
Sorry you ran into this; this command line option is available in 7.0 (which is still in preview) and in the upcoming 6.3 release. We'll update the documentation with these version constraints.

**Comment by LaurenceChau** [comment]
May I know what is the timeline for the 6.3 release? Thanks.

**Comment by jander-msft** [comment]
> May I know what is the timeline for the 6.3 release? Thanks. This feature allows me to set up multiple dotnet-monitor on a machine for exporting different Dotnet app metrics.

I don't have a specific date, but we plan to have it available this month. You can monitor the [releases](https://github.com/dotnet/dotnet-monitor/releases) as it will be published there when it becomes available.

**Comment by jander-msft** [comment]
Fixed by #2607
